src/self-model.scm

```diff
@@ -583,6 +583,7 @@
 (DefineLink
 	(DefinedPredicate "look at person")
 	(SequentialAnd
+		(DefinedPredicate "Time to change gaze")
```

Comment:
@AmeBel I don't know if you are reading these comments or not, but if you are, can you review the above?  The "time-to-change-gaze" should really be in openpsi -- so that if, e.g. 5 or 8 seconds go by, she'll change her gaze, but if someone says "hey look at me know" to the chatbot, she'll change her gaze immediately.